src/lucky/cookies/flash_store.cr

```diff
@@ -0,0 +1,58 @@
+class Lucky::FlashStore
+  SESSION_KEY = "_flash"
+  alias Key = String | Symbol
+
+  delegate each, to: all
+
+  @now = {} of String => String
+  @next = {} of String => String
+
+  def self.from_session(session : Lucky::SessionCookie) : Lucky::FlashStore
+    new.from_session(session)
+  end
+
+  def from_session(session : Lucky::SessionCookie) : Lucky::FlashStore
+    session.get?(SESSION_KEY).try do |json|
+      JSON.parse(json).as_h.each do |key, value|
+        @now[key.to_s] = value.to_s
+      end
+    end
+    self
+  rescue e : JSON::ParseException
```

Comment:
I don’t think we should rescue this like it was before. I feel like if there is an error parsing that should be raised. What do you think?

Reply:
Yeah I think you're right. Raising is better than silent failing. Maybe I could present the contents of the flash in the error?

Reply:
Good idea! I think that would be helpful for debugging

Reply:
Added in https://github.com/luckyframework/lucky/pull/609/commits/ef17cb359d8cc74c48fb026705d4189faaad6131